### ISSUE TITLE
Caseating granuloma

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10128]
**Preferred term label:**
Caseating granuloma

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
HPO has the more granular https://hpo.jax.org/app/browse/term/HP:0033550 but that can't be applied as needed to breast

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Granulomatous lobular mastitis

**Your nano-attribution (ORCID)**
